imports/ethereum/ui/utils/web3.js

```diff
@@ -0,0 +1,12 @@
+import Web3 from 'web3';
+
+let web3 = new Web3();
+
+window.addEventListener('load', function() {
```

Comment:
In Meteor that is Meteor.startup(() => {}) on the client, but ok lets leave it like that maybe we will switch to smth else who knows